src/constants.js

```diff
@@ -1,4 +1,13 @@
+import { io } from 'socket.io-client'
+
 export const userFetchInterval = 3 * 1000
```

Comment:
"_Constants can be declared with uppercase or lowercase, but a common convention is to use all-uppercase letters._ "
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage